client/src_editor/__tests__/Editor_Json_test.re

```diff
@@ -18,14 +18,16 @@ describe("decode", () => {
           "data" : {
             "kind": "code",
             "value": "let a: string = 1;"
-          }
+          },
+          "deleted": false
```

Comment:
you shouldn't change old test json data. old json data should continue work as is.